Netdata assistant launch (#302)
netdata/blog: blog/2023-07-14-netdata-assistant/index.md

<!--truncate-->

## What's the Netdata Assistant?

The Netdata Assistant is an AI tool that uses large language models and our community's knowledge to guide you during troubleshooting.

Here's a scenario. It's 3 am and you get an alert. Instead of scrambling to Google what's going on, you can just click on the assistant button. The Netdata Assistant will give you the lowdown on the alert, why it's happening, and why you should care. It'll also guide you on how to troubleshoot it and even offer some handy web links for more info, if you're interested.

## Why Use Netdata Assistant?

The goal of the Netdata Assistant is straightforward: to make your troubleshooting process easier. It's here to save you from the hassle of sifting through tons of information so you can focus on solving the problem at hand.

And don't worry, we're not about to let AI totally take over. The Netdata Assistant is built to be helpful but not intrusive. You're always in the driver's seat throughout the troubleshooting process.

Let's walk through an example of using Netdata Assistant. 

> 1. This war room has a lot of active alerts, if you want to dive deeper into a particular alert, let's say `load average 15` then just click on the Netdata Assistant button

Reviewer: ```suggestion
1. This war room has a lot of active alerts, if you want to dive deeper into a particular alert, let's say `load average 15`, then just click on the Netdata Assistant button
```

![image](https://github.com/netdata/blog/assets/24860547/815ca123-e2b6-4d44-a780-eeee64cca420)

> 2. Netdata Assistant opens up as a floating window with customized information and troubleshooting tips for this alert

Reviewer: ```suggestion
2. Netdata Assistant opens up as a floating window with customized information and troubleshooting tips for this alert (note that the window can follow you through your troubleshooting journey on Netdata dashboards)
```

![image](https://github.com/netdata/blog/assets/24860547/62850c7b-cd1d-45f2-b2dd-474ecbf2b713)

3. In case you need more information, or want to understand deeper, Netdata Assistant also provides useful web links to resources that can help. This saves you a LOT of time.

![image](https://github.com/netdata/blog/assets/24860547/e768fa6d-6c9a-4504-bb1f-17d5f4707627)
 

## What's Next for Netdata Assistant?

> This is just the start for the Netdata Assistant. We're planning to integrate the assistant into more parts of the Netdata user experience to make your life even easier.

Reviewer: ```suggestion
This is just the start for the Netdata Assistant. We're planning to integrate the assistant into more areas of the Netdata user experience to make your life even easier.
```

I think areas sounds better than parts but feel free to keep the original

> We're pretty excited for you to try out the Netdata Assistant and see how it can help you handle troubleshooting. We're always looking to improve, so don't hesitate to share your experiences with us.

Reviewer: ```suggestion
We're pretty excited for you to try out it and see how it helped you handle troubleshooting. We're always looking to improve, so don't hesitate to share your experiences with us.
```
